[2024-01-18T23:30:30.638+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bitcoin_data_batch_processing.submit_spark_job manual__2024-01-18T23:30:26.571544+00:00 [queued]>
[2024-01-18T23:30:30.645+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bitcoin_data_batch_processing.submit_spark_job manual__2024-01-18T23:30:26.571544+00:00 [queued]>
[2024-01-18T23:30:30.645+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-18T23:30:30.659+0000] {taskinstance.py:1382} INFO - Executing <Task(SparkSubmitOperator): submit_spark_job> on 2024-01-18 23:30:26.571544+00:00
[2024-01-18T23:30:30.661+0000] {standard_task_runner.py:57} INFO - Started process 4617 to run task
[2024-01-18T23:30:30.664+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'bitcoin_data_batch_processing', 'submit_spark_job', 'manual__2024-01-18T23:30:26.571544+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/BatchLayer_dag.py', '--cfg-path', '/tmp/tmpbgkth4e9']
[2024-01-18T23:30:30.667+0000] {standard_task_runner.py:85} INFO - Job 33: Subtask submit_spark_job
[2024-01-18T23:30:30.737+0000] {task_command.py:415} INFO - Running <TaskInstance: bitcoin_data_batch_processing.submit_spark_job manual__2024-01-18T23:30:26.571544+00:00 [running]> on host 36e53310e6a3
[2024-01-18T23:30:30.833+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='bitcoin_data_batch_processing' AIRFLOW_CTX_TASK_ID='submit_spark_job' AIRFLOW_CTX_EXECUTION_DATE='2024-01-18T23:30:26.571544+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-18T23:30:26.571544+00:00'
[2024-01-18T23:30:30.847+0000] {base.py:73} INFO - Using connection ID 'spark_default' for task execution.
[2024-01-18T23:30:30.848+0000] {spark_submit.py:401} INFO - Spark-Submit cmd: spark-submit --master spark://172.29.0.5:7077 --name arrow-spark --queue root.default --deploy-mode client /opt/***/dags/spark_batch.py
[2024-01-18T23:30:30.980+0000] {spark_submit.py:571} INFO - /home/***/.local/lib/python3.9/site-packages/pyspark/bin/load-spark-env.sh: line 68: ps: command not found
[2024-01-18T23:30:33.308+0000] {spark_submit.py:571} INFO - File "/opt/***/dags/spark_batch.py", line 75
[2024-01-18T23:30:33.308+0000] {spark_submit.py:571} INFO - print(¨Charger les données historiques depuis Cassandra¨)
[2024-01-18T23:30:33.309+0000] {spark_submit.py:571} INFO - ^
[2024-01-18T23:30:33.309+0000] {spark_submit.py:571} INFO - SyntaxError: invalid character '¨' (U+00A8)
[2024-01-18T23:30:33.338+0000] {spark_submit.py:571} INFO - 24/01/18 23:30:33 INFO ShutdownHookManager: Shutdown hook called
[2024-01-18T23:30:33.341+0000] {spark_submit.py:571} INFO - 24/01/18 23:30:33 INFO ShutdownHookManager: Deleting directory /tmp/spark-3b0e0986-24e6-44c1-94ab-2f9d7d6234fd
[2024-01-18T23:30:33.378+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 502, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://172.29.0.5:7077 --name arrow-spark --queue root.default --deploy-mode client /opt/***/dags/spark_batch.py. Error code is: 1.
[2024-01-18T23:30:33.382+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=bitcoin_data_batch_processing, task_id=submit_spark_job, execution_date=20240118T233026, start_date=20240118T233030, end_date=20240118T233033
[2024-01-18T23:30:33.399+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 33 for task submit_spark_job (Cannot execute: spark-submit --master spark://172.29.0.5:7077 --name arrow-spark --queue root.default --deploy-mode client /opt/***/dags/spark_batch.py. Error code is: 1.; 4617)
[2024-01-18T23:30:33.415+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-01-18T23:30:33.436+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
